core/client/fs/src/main/java/alluxio/client/file/cache/LocalCacheManager.java

```diff
@@ -371,6 +358,11 @@ public int get(PageId pageId, int pageOffset, int bytesToRead, byte[] buffer,
         "buffer does not have enough space: bufferLength=%s offsetInBuffer=%s bytesToRead=%s",
         buffer.length, offsetInBuffer, bytesToRead);
     LOG.debug("get({},pageOffset={}) enters", pageId, pageOffset);
+    if (mState.get() == NOT_IN_USE) {
+      Metrics.GET_NOT_READY_ERRORS.inc();
+      Metrics.GET_ERRORS.inc();
+      return 0;
```

Comment:
Should this be `return -1` not sure if this counts as error or page not found?

Reply:
updated